webextensions/api/webNavigation.json

```diff
@@ -783,7 +783,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/webNavigation/onTabReplaced",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "22"
```

Comment:
✅ https://github.com/chromium/chromium/commit/b856ad87db6210ebccdbc7365a358ea0de6a44e0 in [22.0.1202.0](https://github.com/chromium/chromium/releases/tag/22.0.1202.0) before 22.0.1229.